src/scripts/main.js

```diff
@@ -1,3 +1,329 @@
 'use strict';
 
-// write your code here
+const button = document.querySelector('.start');
+const messageStart = document.querySelector('.message-start');
+const messageWin = document.querySelector('.message-win');
+const messageLose = document.querySelector('.message-lose');
+const counter = document.querySelector('.game-score');
+
+const rows = document.querySelectorAll('tr');
+const cells = document.querySelectorAll('td');
+
+const tableSize = 4;
+
+const arrayCells = new Array(tableSize);
+
+for (let i = 0; i < arrayCells.length; i++) {
+  arrayCells[i] = new Array(tableSize).fill(0);
+}
+
+let handleKeyDown = false;
```

Comment:
Consider changing the variable name `handleKeyDown` to a more descriptive name, like `canHandleKeyDown` since it is a boolean flag 🚩